### PR TITLE
Add Edinburgh Shows

### PR DESCRIPTION
**Pull request by BenWoodford76**
Adding Dummies and Wrecked

### CODE REVIEW

File: _shows/16_17/escape_for_dummies_edin.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+crew:
+  - role: Director
+    name: Laurence Cuthbert
+  - role: Director
+    name: Josh Mallalieu
+  - role: Producer
+    name: Joanne Blunt
+  - role: Technical Director
+    name: Adam Frankland
+  - role: Publicity manager
```
Comment (nhyty) [inline _shows/16_17/escape_for_dummies_edin.md]: Capitalisation

**Comment by BenWoodford76** [comment]
@samozzy can you check SPAG please?

File: _shows/16_17/escape_for_dummies_edin.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+links:
+  - type: Review
+    href: http://edinburgh.fringeguru.com/reviews/edinburgh-2017/escape-for-dummies
+    snapshot: Ju6HL
+    publisher: FringeGuru
+    author: Richard Stamp
+    title: "Escape for Dummies"
+    date: 2017-08-19
+    stars: 5
+    quote: "Escape For Dummies is an hour of pure joy – hilarious and giddy, yet with characters who make you care. Plan ahead for this one, because it's performing on odd-numbered dates only… and you'd have to be a dummy to miss it."
```
Comment (samozzy) [inline _shows/16_17/escape_for_dummies_edin.md]: Would shorten the end to:
`characters who make you care. Plan ahead for this one [...] you'd have to be a dummy to miss it.`

File: _shows/16_17/escape_for_dummies_edin.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    quote: "This play is a unique and hidden gem at the Fringe. It’s a rare show in that it is suitable for the entire family, as well as for deaf audience members."
+  - type: Review
+    href: http://edfringereview.com/r/WYh34KJATEyhgk0jXDx6yg
+    snapshot: 5Eb4u
+    publisher: edfringereview
+    author: James Tibbles
+    title: "Escape for Dummies"
+    date: 2017-08-20
+    stars: 5
+    quote: "Escape for Dummies comprises an ensemble of five who all embody distinct caricatures in a masterclass of physical comedy. A simple escape plot sees two manikins come to life around a myriad of silent stunts and scenes that has the audience laughing from the very first moment."
```
Comment (samozzy) [inline _shows/16_17/escape_for_dummies_edin.md]: Should we add a `[sic]` after `manikins`, as this is slightly different to the mannequins the actors actually play?

File: _shows/16_17/wrecked_edin.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+assets:
+  - type: poster
+    image: t6mfPwb
+
+links:
+  - type: Review
+    href: http://edfringereview.com/r/WYh06bDwTo6x5kwRXDx6yg?
+    snapshot: fKFbP
+    publisher: edfringereview
```
Comment (samozzy) [inline _shows/16_17/wrecked_edin.md]: Where should spacing be? The website header has it as 'edfringe review' while the about page has 'Ed Fringe Review'

Comment (wjdp) [reply _shows/16_17/wrecked_edin.md]: Also take that `?` out

**Pull request merged**